src/index.html

```diff
@@ -4,9 +4,329 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="Moyo logo">
```

Comment:
```suggestion
        <img
          class="header__logo-img"
          src="./images/logo.png"
          alt="Moyo logo"
        >
```
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/bbc3057e-3295-4358-9d0d-0bbd9149458b)